python/packages/core/agent_framework/openai/_chat_client.py

```diff
@@ -369,8 +369,6 @@ def _openai_chat_message_parser(self, message: ChatMessage) -> list[dict[str, An
             args: dict[str, Any] = {
                 "role": message.role.value if isinstance(message.role, Role) else message.role,
             }
-            if message.additional_properties:
```

Comment:
The anthropic chat client seems to be adding all `additional_properties` to `run_client` - 

```py
if chat_options.additional_properties:
            run_options.update(chat_options.additional_properties)
```

I wonder if we should remove it here too?